### ISSUE TITLE
transfer_learning_tutorial. 缺少参数：dataloaders

### ISSUE THREAD

**Issue by my-hello-world** [apachecn/pytorch-doc-zh#385]
https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/transfer_learning_tutorial.mddataloaders

def train_model_scheduler(model, dataloaders,criterion, optimizer, scheduler, num_epochs=5,):
